docsite/components/modal.html

```diff
@@ -0,0 +1,146 @@
+                        <button class="reset padding-sm" id=
+                        "cancel-btn-sm">Cancel</button>
+                       </div>
+                </div>
+            </div>
+                <div class="iframe-wrapper">
+                    <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%28155%2C155%2C155%2C1%29&t=night-owl&wt=sharp&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253C%21--HTML%2520--%253E%250A%253Cdiv%2520class%253D%2522%2520centered%2520shadow-box%2520margin-md%2520padding-y%2522%253E%2520%250A%2509%253Cbutton%2520class%253D%2522padding-sm%2520reset%2520btn-outline%2520font3%2522%2520id%253D%2522modal-sm-popup-btn%2522%253EShow%2520me%253C%252Fbutton%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522modal%2520modal-sm%2520shadow-box%2520mb-lg%2522%2520id%253D%2522sm-modal%2522%253E%250A%2520%2520%2509%253Cdiv%2520class%253D%2522font3%2520padding-xs%2522%253ESimple%2520small%2520modal%253C%252Fdiv%253E%253Chr%252F%253E%250A%2520%2520%2520%2520%253Cp%2520class%253D%2522text-vibrant%2520padding-xs%2520mb-lg%2522%253ELorem%2520ipsum%252C%2520dolor%2520sit%2520amet%2520consectetur%2520adipisicing%2520elit.%2520%250A%2520%2509Eligendi%2520harum%2520odit%2520quasi.%253C%252Fp%253E%250A%2520%2520%2520%253Cbutton%2520class%253D%2522reset%2520padding-sm%2522%2520id%253D%2522cancel-btn-sm%2522%253ECancel%253C%252Fbutton%253E%250A%2520%2520%2520%2509%2520%253C%252Fdiv%253E%250A%2520%253C%252Fdiv%253E%250A%253C%21--js--%253E%250A%250Avar%2520btnsmallModal%2520%253D%2520document.querySelector%28%2522%2523modal-sm-popup-btn%2522%29%250Avar%2520smallModal%2520%253D%2520document.querySelector%28%2522%2523sm-modal%2522%29%250Avar%2520cancelbtnsm%2520%253D%2520document.querySelector%28%2522%2523cancel-btn-sm%2522%29%250AbtnsmallModal.addEventListener%28%2522click%2522%252C%28%29%253D%253E%257B%250A%2520%2520%2520%2520smallModal.style.display%253D%2522block%2522%250A%257D%29%250Acancelbtnsm.addEventListener%28%2522click%2522%252C%28%29%253D%253E%257B%250A%2520%2520%2520%2520smallModal.style.display%2520%253D%2522none%2522%250A%257D%29"
+  class="iframe-lg"
+  sandbox="allow-scripts allow-same-origin">
+</iframe>
+                </div>
+                
+            </div>
+           <div class="section">
+            <h2 class="mb-lg" >Login modal</h2>
+            <p class="mb-lg font3 text-secondary">Login modals can be used for user login or signin.</p>
+            <div class=" centered shadow-box margin-md padding-y">
+               <button class="padding-sm reset rm-border bg-secondary light-text font3" id="modal-lg-popup-btn">Login</button>
+               <div class="modal modal-lg" id="login-modal">
+                <div class="modal-header reset " id="signup-close-btn">X</div>
```

Comment:
Just a suggestion, use icon instead of X.

Reply:
Ok I will take care of that from now